astroquery/cadc/tests/DummyJob.py

```diff
@@ -0,0 +1,30 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+=============
+Cadc TAP plus
+=============
+
+"""
+
+
+class DummyJob(object):
```

Comment:
Same comment as above: try to use Mock next time.

Reply:
I think astroquery likes it that way